app/controllers/appliances_controller.rb

```diff
@@ -25,7 +26,7 @@ def edit
   # POST /appliances
   # POST /appliances.json
   def create
-    @appliance = Appliance.new(appliance_params)
+    @appliance = Appliance.create(appliance_params)
```

Comment:
Why did you changed `new `to `create`?
This not advised because you save `@appliance` again in this action.

Reply:
True, thanks for pointing out!